src/index.js

```diff
@@ -23,7 +24,7 @@ const middleware = async (req, res, next) => {
   switch (req.path) {
     case heapWebPath:
       try {
-        await heap(heapFilePath);
+        await heap(heapFilePath, pprof);
```

Comment:
Why pass the module as a param?

Reply:
@lazorfuzz  Passing the `pprof` instance to both the `heap` and `wall` modules ensures they share the same instance, guaranteeing consistency and avoiding potential issues that may arise from using different instances of `pprof`. This approach utilizes a single instance of `pprof` for efficiency and consistency reasons.

Reply:
@lazorfuzz  Passing the `pprof` instance to both the `heap` and `wall` modules ensures they share the same instance, guaranteeing consistency and avoiding potential issues that may arise from using different instances of `pprof`.

Reply:
Hey @lazorfuzz can you review this PR and give some suggestion / feedback?